nixos/doc/manual/configuration/profiles/minimal.section.md

```diff
@@ -1,6 +1,6 @@
 # Minimal {#sec-profile-minimal}
 
 This profile defines a small NixOS configuration. It does not contain any
-graphical stuff. It's a very short file that sets [](#opt-i18n.supportedLocales)
```

Comment:
I thought this was a change that should be decoupled from the `visible = false;` change.